src/hooks/useOutsideClick/useOutsideClick.test.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { useRef } from 'react';
+import { fireEvent, render } from '@testing-library/react';
+import useOutsideClick from './useOutsideClick';
+
+const handler = jest.fn();
+function ClickableComponent(): JSX.Element {
```

Comment:
We are working in ES6+, Could we change it into functional component using ES6 syntax?